Makefile

```diff
@@ -99,8 +84,8 @@ fmt:
 	@gofmt -l -s -w $(SOURCE_DIRS)
 
 validate: lint
-	@go vet $(PACKAGES)
-	@test -z "$$(gofmt -s -l . | grep -ve '^vendor' | tee /dev/stderr)"
```

Comment:
Please keep the `vendor` filter. We might have checked out the code locally via `go mod vendor`.

Reply:
Done!